etcdserver/api/membership/member.go

```diff
@@ -51,10 +53,13 @@ type Member struct {
 
 // NewMember creates a Member without an ID and generates one based on the
 // cluster name, peer URLs, and time. This is used for bootstrapping/adding new member.
-func NewMember(name string, peerURLs types.URLs, clusterName string, now *time.Time) *Member {
+func NewMember(name string, peerURLs types.URLs, clusterName string, now *time.Time, isLearner bool) *Member {
```

Comment:
Feels like at some point we should replace the positional parameters with a options struct that is passed in so we don't have to go change all the call sites when adding a new param with a reasonable default.

Reply:
Agreed.